tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -45,6 +40,19 @@
     </ItemGroup>
   </Target>
 
+  <!-- 
+    In the above AddSingleConfigBuildOutput target, several properties are injected into this project so that this project knows if its being multiplexed or not. Don't pass on this extra information to project references as it can issues with MSBuild knowing if a project is already built.
+  -->
+  <Target Name="RemoveGlobalPropertiesFromMultiplexing" BeforeTargets="BeforeResolveReferences">
+    <_GlobalPropertiesToRemove>BuildingPackage;</_GlobalPropertiesToRemove>
+    <_GlobalPropertiesToRemove Condition="'$(SkipGetPackageContentsForOtherPlatforms)' == 'true'">$(_GlobalPropertiesToRemove);SkipGetPackageContentsForOtherPlatforms;IsPackable;</_GlobalPropertiesToRemove>
```

Comment:
Because you wrote Packager and I wrote common? You pass through IsPackable on line #22 whereas I keyed off of the SkipGetPackageContentsForOtherPlatforms to set it in a props. Its not a big deal either way we just need to account for it.

---
In reply to: [110029872](https://github.com/Microsoft/WinObjC/pull/2422#discussion_r110029872) [](ancestors = 110029872)